content/2021-12-01-drogue-cloud-zero-eight/index.md

```diff
@@ -0,0 +1,142 @@
+MQTT endpoint got extended to support this.
+
+## Static server binary
+
+Deploying a complex, cloud native stack like Drogue Cloud can be tricky. We put a lot of effort in our installer scripts
+and Helm charts to make it both easy and customizable. However, every now and then, it would be nice to "just run"
+Drogue Cloud. With 0.8, you can!
+
+Assuming you have an existing Kafka and Keycloak instance, it is as easy as:
+
+~~~shell
+drogue-cloud-server --enable-all
+~~~
+
+And, it also works on Apple M1 machines. Does it bring all the features? No, it can't. There are a few that we will
+add in future releases, but we are not planning to re-invent Kubernetes. Does it run on a Raspberry Pi too?
+Probably, we didn't have the time yet to try out. But we definitely want to get there.
+
+If you are curious, you can read more about it
+[in our documentation](https://book.drogue.io/drogue-cloud/dev/deployment/bare-metal.html).
+
+## More workshops
+
+The idea behind out workshops is to give you concrete examples on how to use Drogue Device and Drogue Cloud. And to
+enable you, to replicate this on your own. And since the last release, we added two more workshops to our list.
+
+### Eclipse IoT package – Telemetry end-to-end
+
+The first one was in the making for quite some time. It shows the integration of a handful of IoT related open source
+projects from Eclipse IoT. And so this is in fact not a Drogue IoT workshop, but an Eclipse IoT package
+named [Telemetry end-to-end](https://www.eclipse.org/packages/packages/telemetry-e2e/).
+
+It showcases the use case of telemetry data acquisition, from a Drogue Device based microcontroller,
+through an Eclipse Kura IoT gateway, into Drogue Cloud. On the cloud side we have Eclipse Ditto as digital twin platform
+and Eclipse Streamsheets to visualize the data. All you need is a micro:bit v2, a Raspberry Pi, and your local machine.
+
+It also showcases how important the ability to integrate is. All these different projects, developed by different
+communities, can still work together to solve problems on the different layers of an IoT stack. Try it out, and maybe
+replace a component yourself.
+
+Of course, you can also cheat and just focus on the cloud side running on your laptop. Should you want to skip on buying
+a few new toys to play with.
+
+### Drogue WiFi
+
+This examples walks you through a set of steps, creating your own microcontroller firmware, measuring a temperature,
+and sending that to the cloud. Heard that before? Well sure, not only is this our prime example for the use
+case of sending telemetry data to the cloud, we also have a few other examples doing the same. Just not one that is a
+complete workshop, including a WiFi based firmware.
+
+On the cloud side, the workshop focuses on re-using an existing Drogue cloud instance. You still host the cloud
+application, the part which consumes the device data, yourself. And as always, you are also able to run Drogue Cloud
+yourself.
```

Comment:
```suggestion
There are two examples of how to consumer data using web sockets, a custom WebAssembly application and ready-to-use TimescaleDB/Grafana dashboard. They represent a good starting point for developing IoT applications based on Drogue Cloud
```

Reply:
In think the suggestion is off a bit. Please check if I applied that correctly.

Reply:
Looks good